studybuddies/src/app/student/class-list-student/class-list-student.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ClassListStudentComponent } from './class-list-student.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155411)
```suggestion
import { ClassListStudentComponent } from "./class-list-student.component";
```